src/Pages/Main/Main.js

```diff
@@ -2,8 +2,51 @@ import React from "react";
 import "./Main.scss";
 
 class Main extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      mainData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/commonData.json")
+      .then((res) => {
+        return res.json();
+      })
+      .then((result) => {
+        this.setState({
+          mainData: result.data,
+        });
+      });
+  }
   render() {
-    return <div>Main</div>;
+    const { mainData } = this.state;
+    console.log(mainData);
+    return (
+      <div className="Main">
+        <div className="mainContent">
+          <ul>
+            {mainData.length &&
+              mainData.map((el, i) => {
+                return (
+                  <li>
+                    <img alt={`mainImage${i}`} src={el.url} />
+                    <div className="textBox">
+                      <span className="title">{el.title}</span>
+                      <div className="contentBox">
+                        <p className="contentOne">{el.contentOne}</p>
+                        <p className="contentTwo">{el.contentTwo}</p>
+                      </div>
```

Comment:
```suggestion
            {mainData?.map((el, i) => {
                return (
                  <li key={el.url}>
                    <img alt={`mainImage${i}`} src={el.url} />
                    <div className="textBox">
                      <span className="title">{el.title}</span>
                      <div className="contentBox">
                        <p>{el.contentOne}</p>
                        <p>{el.contentTwo}</p>
                      </div>
```
- 옵셔널 체이닝을 이용해서 조금 더 간결하게 써줄 수 있습니다. 옵셔널 체이닝은 뒤의 값이 유효한지를 판단하여, 참일 때만 실행하게 됩니다. 링크 한번 참고해보세요! 
링크 : https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining
- 클래스명 없이도 p 태그의 순서에 따라 스타일링 할 수 있을듯 합니다.